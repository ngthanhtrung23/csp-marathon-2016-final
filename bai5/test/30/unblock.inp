122234
155534
0067.4
..67..
...899
...8AA
