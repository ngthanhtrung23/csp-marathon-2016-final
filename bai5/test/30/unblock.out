29
6 D 1
0 R 1
1 D 3
0 L 1
2 L 1
5 L 1
6 U 1
7 U 2
8 U 2
9 L 3
3 D 3
A L 3
4 D 3
8 D 2
7 D 2
2 R 3
5 R 3
6 U 2
0 R 1
1 U 3
0 L 1
9 L 1
6 D 3
0 R 1
1 D 2
2 L 3
5 L 3
7 U 2
0 R 3

